django_project/lesson/forms/question.py

```diff
@@ -32,7 +33,11 @@ def __init__(self, *args, **kwargs):
             Fieldset(
                 _('Worksheet {}: question details').format(self.worksheet),
                 Field('question', css_class='form_control'),
-                Field('question_image', css_class='form_control'),
+                HTML(
+                    '{% if form.question_image.value %}'
+                    '<img class="img-responsive" '
+                    'src="{{ MEDIA_URL }}{{ form.question_image.value }}">'
```

Comment:
Can't be replaced by form.question_image.url?
I saw it was better to use the `url` property instead of string concatenation.

Reply:
I tried to use `form.question_image.url` but it gives empty string.